testsuite/features/support/remote_node.rb

```diff
@@ -361,4 +362,12 @@ def get_os_version(runs_in_container: true)
     $stdout.puts "Node: #{@hostname}, OS Version: #{os_version}, Family: #{os_family}"
     [os_version, os_family]
   end
+
```

Comment:
`host` and `container` must be defined above the `private` keyword, otherwise they are not public and accessible from outside.

Reply:
You can see the issue on the tests:
```
private method `host' called for an instance of RemoteNode (NoMethodError)
/testsuite/features/support/commonlib.rb:296:in `transactional_system?'
/testsuite/features/support/env.rb:70:in `<top (required)>'
```

Example https://github.com/uyuni-project/uyuni/actions/runs/14165727780/job/39678504110?pr=10092 at "Run Core tests"

Reply:
Thanks, I have moved the `host` and `container` to the above the `private`. Please check it.